src/components/AnalysisPage/3dModelViewer/3dModelViewer.jsx

```diff
@@ -30,6 +31,15 @@ const ModelViewer = ({ fileFor3dModel }) => {
     return () => cleanup();
   }, [objectURL]);
 
+  useEffect(() => {
+    const handleResize = () => {
+      forceContainerResize((prev) => !prev);
```

Comment:
I dont really understand this. What does forceContainerResize do? Is it just placeholder code for the future? Doesn't really seem like its doing anything currently

Reply:
It was making the model viewer responsive at some point in my endeavour. But apparently at some point along the way, I adjusted things enough so that it is no longer doing anything.

I wanted to add it in its own commit, so i removed it, and then added it after the fact without double checking that it was still actually needed. I may have even wrote it wrong the second time.

I'll remove it!

Reply:
I need to remove that merge commit before pushing this change. I'm about to go back into work, we'll have to do this later!